plugins/copy-by-storage/src/index.js

```diff
@@ -0,0 +1,243 @@
+        localStorage.setItem('blocklyStash', blockText);
+        return true;
+      },
+    };
+    Blockly.ShortcutRegistry.registry.register(copyShortcut);
+
+    const ctrlC = Blockly.ShortcutRegistry.registry.createSerializedKey(
+        Blockly.utils.KeyCodes.C, [Blockly.utils.KeyCodes.CTRL]);
+    Blockly.ShortcutRegistry.registry.addKeyMapping(ctrlC, copyShortcut.name);
+
+    const altC =
+      Blockly.ShortcutRegistry.registry.createSerializedKey(
+          Blockly.utils.KeyCodes.C, [Blockly.utils.KeyCodes.ALT]);
+    Blockly.ShortcutRegistry.registry.addKeyMapping(altC, copyShortcut.name);
+
+    const metaC = Blockly.ShortcutRegistry.registry.createSerializedKey(
+        Blockly.utils.KeyCodes.C, [Blockly.utils.KeyCodes.META]);
+    Blockly.ShortcutRegistry.registry.addKeyMapping(metaC, copyShortcut.name);
+  }
+
+  /**
+   * Set the cut command to keybord shortcut
+   */
+  blockCutToStorageShortcut() {
+    /** @type {!Blockly.ShortcutRegistry.KeyboardShortcut} */
+    const cutShortcut = {
+      name: 'cut',
+      preconditionFn: function(workspace) {
+        return !workspace.options.readOnly &&
+          !Blockly.Gesture.inProgress() &&
+          Blockly.selected &&
+          Blockly.selected.isDeletable() &&
+          Blockly.selected.isMovable() &&
+          !Blockly.selected.workspace.isFlyout;
+      },
+      callback: function(workspace, e) {
+        // Prevent the default copy behavior,
+        // which may beep or otherwise indicate
+        // an error due to the lack of a selection.
+        e.preventDefault();
+        const blockText = JSON.stringify(
+            convertBlockToSaveInfo(Blockly.selected));
+        localStorage.setItem('blocklyStash', blockText);
+        Blockly.Events.setGroup(true);
+        Blockly.selected.dispose(true);
+        Blockly.Events.setGroup(false);
+        return true;
+      },
+    };
+    Blockly.ShortcutRegistry.registry.register(cutShortcut);
+
+    const ctrlX = Blockly.ShortcutRegistry.registry.createSerializedKey(
+        Blockly.utils.KeyCodes.X, [Blockly.utils.KeyCodes.CTRL]);
+    Blockly.ShortcutRegistry.registry.addKeyMapping(ctrlX, cutShortcut.name);
+
+    const altX = Blockly.ShortcutRegistry.registry.createSerializedKey(
+        Blockly.utils.KeyCodes.X, [Blockly.utils.KeyCodes.ALT]);
+    Blockly.ShortcutRegistry.registry.addKeyMapping(altX, cutShortcut.name);
+
+    const metaX = Blockly.ShortcutRegistry.registry.createSerializedKey(
+        Blockly.utils.KeyCodes.X, [Blockly.utils.KeyCodes.META]);
+    Blockly.ShortcutRegistry.registry.addKeyMapping(metaX, cutShortcut.name);
+  }
+
+  /**
+   * Set the paste command to keybord shortcut
+   */
+  blockPasteFromStorageShortcut() {
+    /** @type {!Blockly.ShortcutRegistry.KeyboardShortcut} */
+    const pasteShortcut = {
+      name: 'paste',
+      preconditionFn: function(workspace) {
+        if (workspace.options.readOnly || Blockly.Gesture.inProgress()) {
+          return 'disabled';
```

Comment:
The keyboard shortcut precondition should only return `true` or `false` values. Can you update the 'disabled' to be false and the 'enabled' to be true in this method?